dags/reporting_dag.py

```diff
@@ -0,0 +1,183 @@
+# from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.providers.postgres.operators.postgres import PostgresOperator, PostgresHook
+from airflow.utils.task_group import TaskGroup
+from airflow.providers.google.cloud.operators.gcs import GCSListObjectsOperator
+from datetime import datetime, timedelta
+import json
+from operators.airflow_to_gcs import AirflowToGCSOperator
+from operators.gcs_to_postgres import GCSToPostgres
+from airflow.decorators import dag, task
+
+
+@dag(
+    start_date=datetime(2019, 1, 1),
+    max_active_runs=3,
+    schedule_interval=None,
+    catchup=False,
+    template_searchpath=f"include/sql/",
+)
+def reporting_dag():
+    pg_hook = PostgresHook(postgres_conn_id="my_postgres_conn_id")
```

Comment:
Might be worth it to have a `postgres_conn_id` variable at the top to pass in where this string is repeated.